src/PKSim.Core/Model/ImportPopulationFactory.cs

```diff
@@ -135,39 +132,11 @@ private Task<ImportResult> importFiles(string file, CancellationToken cancellati
          }, cancellationToken);
       }
 
-      private void updatePathsContaininUnits(IndividualPropertiesCache individualValues)
```

Comment:
THis code can be used as is. It's just in the wrong place. It was done in the `ImportPopulationFactory` instead of the `ImportFile`